CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
-        public Guid CompanyId { get; set; }
+public decimal Salary { get; set; }
```

Comment:
The new `Salary` field was added before the `Company` navigation property, which may visually separate related fields (`CompanyId`, `Company`). Consider reordering for clarity.

```suggestion
public decimal Salary { get; set; }
```